pkg/cmd/cmd.go

```diff
@@ -75,6 +75,12 @@ func RunContext(ctx context.Context, name string, args ...string) (string, error
 	}
 
 	if err = reaper.WaitWrapper(usingReaper, notifyCh, cmd); err != nil {
+		if e, ok := err.(*reaper.ExitError); ok { //nolint:errorlint
+			e.Output = stderr.Bytes()
```

Comment:
I feel it's better to wrap the error here if needed instead of trying to modify error value on the fly